gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,7 +1,6 @@
-#Thu Sep 14 09:54:35 CEST 2017
+#Fri Feb 16 18:55:01 CET 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.14.1-all.zip
-distributionSha256Sum=88a910cdf2e03ebbb5fe90f7ecf534fc9ac22e12112dc9a2fee810c598a76091
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.0.2-bin.zip
```

Comment:
Why not 4.5.1?

Reply:
`distributionSha256Sum` is missing

Reply:
>Why not 4.5.1?

Good question! We don't want to force clients to latest Gradle, we want to allow them to stay at 4.2 or 4.3.

Reply:
That's why I created regression tests with older Gradle version(s) long time ago. Thanks to that the project can be built with the latest version while (to the extent covered by tests :) ) it is still compatible with older versions.